drivers/usb/usbxhci/usbxhci.c

```diff
@@ -0,0 +1,572 @@
+/*
+ * PROJECT:     ReactOS USB xHCI Miniport Driver
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     USBXHCI main driver functions
+ * COPYRIGHT:   Copyright 2023 Ian Marco Moffett <ian@vegaa.systems>
+ */
+
+#include "usbxhci.h"
+
+#define NDEBUG
+#include <debug.h>
+
+#define XHCI_POLL_TIME_SET(Milliseconds) (Milliseconds * 10000)
+
+/* Status codes for XHCI_PollCheck() */
```

Comment:
You could use NTSTATUS codes like `STATUS_SUCCESS` / `STATUS_PENDING` / `STATUS_TIMEOUT` here, instead of defining a new enum. But either way is fine with me.